src/pages/SignIn/components/SignInForm/index.js

```diff
@@ -1,73 +1,53 @@
-import React from "react";
-import InputAdornment from "@mui/material/InputAdornment";
-import TextField from "@mui/material/TextField";
-import PersonIcon from "@mui/icons-material/Person";
-import MailOutlineIcon from "@mui/icons-material/MailOutline";
-import HttpsIcon from "@mui/icons-material/Https";
-import {
-  FormWrapper,
-  FormTitle,
-  FormSubTitle,
-  FormButton,
-  FormText,
-} from "./styled";
+import React from 'react';
+import InputAdornment from '@mui/material/InputAdornment';
+import TextField from '@mui/material/TextField';
+import PersonIcon from '@mui/icons-material/Person';
+import MailOutlineIcon from '@mui/icons-material/MailOutline';
+import HttpsIcon from '@mui/icons-material/Https';
+import { FormWrapper, FormTitle, FormSubTitle, FormButton, FormInner } from './styled';
 
 const SignInForm = () => {
   return (
     <FormWrapper>
-      <FormText>
-        <FormTitle variant="h3">Hello</FormTitle>
-        <FormSubTitle variant="h4">Sign Up to Get Started</FormSubTitle>
-      </FormText>
+      <FormInner>
+        <FormTitle variant='h3'>Hello</FormTitle>
+        <FormSubTitle variant='h4'>Sign Up to Get Started</FormSubTitle>
 
-      <TextField
-        placeholder="Full Name"
-        sx={{ m: 1, width: "400px" }}
-        InputProps={{
-          startAdornment: (
-            <InputAdornment position="start">
-              <PersonIcon />
-            </InputAdornment>
-          ),
-        }}
-      />
+        <TextField
+          placeholder='Full Name'
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position='start'>
+                <PersonIcon />
+              </InputAdornment>
+            ),
+          }}
+        />
 
-      <TextField
-        placeholder="Email"
-        sx={{ m: 1, width: "400px" }}
-        InputProps={{
-          startAdornment: (
-            <InputAdornment position="start">
-              <MailOutlineIcon />
-            </InputAdornment>
-          ),
-        }}
-      />
+        <TextField
+          placeholder='Email'
+          sx={{ my: '15px' }}
```

Comment:
No!
Remove `sx={{ my: '15px' }}`

What if you will have 10 fields?

you have to use Stack component from MUI - https://mui.com/material-ui/react-stack/#interactive

<img width="713" alt="Screenshot 2023-01-03 at 22 39 32" src="https://user-images.githubusercontent.com/13525860/210445714-91b42a67-0161-4b89-a5bd-3464b893be18.png">

Reply:
Fixed